app/views/diary_entries/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= render GovukComponent::SummaryList.new do |component| %>%
```

Comment:
Unless it was explicitly asked for, I would not put the `show` page in, until it's decided how it'll behave and what it looks like?

Reply:
I think we have to take some initiative, there will most likely be a show page, as the entries are truncated on the index list, and might be a good feature for the Show and Tell